scripts/commands/update-translations.js

```diff
@@ -52,9 +52,12 @@ export default async function updateTranslations() {
 			`wp i18n make-pot . ${ POT_FILE_PATH } --ignore-domain`
 		);
 
+		// If execPromise doesn't reject the promise, the command was successful
```

Comment:
I would be tempted to move this comment to just before the `execPromise` call. It might also be noting in the comment that, if `execPromise` fails, an exception will be thrown.